.github/workflows/ci.yml

```diff
@@ -21,9 +21,6 @@ jobs:
       - name: Run black formatter
         run: black --check .
 
-      - name: Run isort
```

Comment:
why we removed sorted imports check?

Reply:
Most of mitodl repos use [black linter](https://pypi.org/project/black/) which I didn't found compatible with [isort](https://pypi.org/project/isort/). I tried skipping isort on the specific clash points but found that sort doesn't obey `#isort:skip`; `#isort:off` as it should. So without investing too much time investigating it I thought it's better to stay consistent with all mitodl repos and use the black linter and pylint which also provide import sorting features.